discretisedfield/plotting/pyvista_field.py

```diff
@@ -0,0 +1,770 @@
+import numpy as np
+import pyvista as pv
+import ubermagutil.units as uu
+
+import discretisedfield.plotting.util as plot_util
+
+
+class PyVistaField:
+    def __init__(self, field):
+        if field.mesh.region.ndim != 3:
+            raise RuntimeError("Only 3d meshes can be plotted.")
+        self.field = field * 1
```

Comment:
Why do we need the multiplication with one?

Edit: is it used to get a copy? If yes I suggest we implement a proper copy method instead.

Reply:
I have implemented a copy method for now. I have made an issue #516 for us to do this. For now I suggest we resolve this and then change the code when we have written the method.